### PR TITLE
Feature: Carthage support

### PR DESCRIPTION
**Pull request by 3lvis**
## Pull request checklist

- [x] All tests pass. 
- [x] Demo project builds and runs.
- [x] I have resolved merge conflicts.
- [x] I have followed the [coding style](https://github.com/jessesquires/HowToContribute#style-guidelines). 

[Contributing guidelines](https://github.com/jessesquires/JSQMessagesViewController/blob/develop/.github/CONTRIBUTING.md) confirmation: Confirmed.

## What's in this pull request?

Adds Carthage support

### CODE REVIEW

File: JSQMessagesViewController.podspec [hunk 1]
```diff
@@ -1,10 +1,10 @@
 Pod::Spec.new do |s|
 	s.name = 'JSQMessagesViewController'
-	s.version = '7.3.4'
+	s.version = '8.0.0-beta1'
 	s.summary = 'An elegant messages UI library for iOS.'
 	s.homepage = 'http://jessesquires.github.io/JSQMessagesViewController'
 	s.license = 'MIT'
-	s.platform = :ios, '7.0'
+	s.platform = :ios, '8.0'
```
Comment (3lvis) [inline JSQMessagesViewController.podspec]: >Note that Carthage only supports dynamic frameworks, which are only available on iOS 8 or later (or any version of OS X).

File: JSQMessagesViewController/JSQMessages.h [hunk 2]
```diff
@@ -22,6 +22,11 @@
 #import "JSQMessagesViewController.h"
 
 //  Views
+#import "JSQMessagesCellTextView.h"
```
Comment (3lvis) [inline JSQMessagesViewController/JSQMessages.h]: These were missing from the original umbrella header.

**Comment by geoherna** [comment]
So is Carthage actually supported now? Adding `github "jessesquires/JSQMessagesViewController"` to Cartfile but getting `Dependency "JSQMessagesViewController" has no shared framework schemes for any of the platforms: iOS`

**Comment by ikzjfr0** [comment]
i got this issue too. please clarify

**Comment by jessesquires** [comment]
Hey all -- sorry for the delay here. 😅 

We can't support carthage yet (in 7.x releases and prior) due to some legacy issues.

This is an area of focus for the 8.0 release.

**Pull request closed without merging**